globalsat_sport.cc

```diff
@@ -0,0 +1,742 @@
+/*
+    Globalsat Sport serial protocol.
+
+    Copyright (C) 2012, Zingo Andersen
```

Comment:
And me, please, for legal reasons.  It's easier to defend in a copyright tussle if I'm an owner.